UEBPrep/app/models/user.rb

```diff
@@ -62,6 +62,7 @@ def self.find_for_twitter_oauth(auth, signed_in_resource=nil)
 
         user = User.create(provider:auth.provider,
                            uid:auth.uid,
+                           username:auth.uid+"@twitter.com",
```

Comment:
right. see my comment above. there is currently no validation for the username field. If you want to add that story I will do that as well.